shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/statement/impl/MySQLDALStatementSQLVisitor.java

```diff
@@ -238,7 +243,57 @@ public ASTNode visitShowWarnings(final ShowWarningsContext ctx) {
     
     @Override
     public ASTNode visitResetStatement(final ResetStatementContext ctx) {
-        return new MySQLResetStatement();
+        MySQLStatementParser.ResetPersistContext persistContext = ctx.resetPersist();
+        if (persistContext != null) {
+            return visit(persistContext);
+        }
+        MySQLResetStatement result = new MySQLResetStatement();
+        List<MySQLStatementParser.ResetOptionContext> opts = ctx.resetOption();
+        for (MySQLStatementParser.ResetOptionContext optCtx : opts) {
```

Comment:
@flycash Do you think it is better to use `ctx.resetOption()` instead of `opts`? Because it was only used once.

Reply:
yes, corrected